src/Ilios/CoreBundle/Tests/Entity/EntityBase.php

```diff
@@ -20,9 +24,9 @@ public function tearDown()
     }
 
     /**
-     * Engage the symfony validator and test the object
-     * @param  integer $expectedCount how many erors are you expecting
-     * @return array an abreviated set of errors
+     * Engage the symfony validator and test the object.
+     * @param  integer $expectedCount how many errors are you expecting
+     * @return array an abbreviated set of errors
```

Comment:
I read this like three times trying to figure out why you were changing it.  Then I realized it was just my terrible spelling.